aptos-move/framework/aptos-framework/sources/native_bridge.move

```diff
@@ -97,21 +136,65 @@ module aptos_framework::native_bridge {
 
     struct BridgeConfig has key {
```

Comment:
the risk denominator should be explained as it is not obvious from the name what it is for. Here some proposed text

> The risk_denominator determines the fraction of the insurance fund that can be used for the per day budget for a given transfer direction.